views/extras.tmpl

```diff
@@ -230,8 +230,8 @@
       </thead>
       <tbody>
           {{range $i, $v := .Transactions}}
+          {{with $v}}
           <tr {{if eq .Confirmations 0}} data-addr-tx-pending="{{.TxID}}" {{end}}>
```

Comment:
Seems like `with $v` had to be before the `tr` line referencing `.Confirmations` and `.TxID`...